### PR TITLE
travis: upgrade distribution to Ubuntu 20.04 …

### PR DESCRIPTION
**Pull request by marc-hb**
The current default Travis default is 16.04 which is obsolete.

From thesofproject/linux#2402

> Everyone uses 20.04, period. SOF CI also uses 20.04.

Everything used Docker anyway.

Signed-off-by: Marc Herbert <marc.herbert@intel.com>

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@xiulipan any comments ?

File: .travis.yml [hunk 1]
```diff
@@ -49,13 +49,15 @@ jobs:
...
+        - sed -i $(($(grep "config HAVE_AGENT" -n src/platform/Kconfig |
+             cut -f1 -d:)+2))"s/default y/default n/" src/platform/Kconfig
         - ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -r $PLATFORM
         - ./scripts/docker-qemu.sh ../sof.git/scripts/qemu-check.sh $PLATFORM
       env: PLATFORM='byt cht'
       before_install:
         - *docker-pull-sof
-        - docker pull thesofproject/sofqemu && docker tag thesofproject/sofqemu sofqemu
+        - docker pull thesofproject/sofqemu &&
+            docker tag thesofproject/sofqemu sofqemu
```
Comment (paulstelian97) [inline .travis.yml:62]: Are we folding based on the 80-character lines or the 100-character lines?

**Review by xiulipan — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (marc-hb) [reply .travis.yml:62]: The (other...) commit message says: "fix the last yamllint warnings".

Comment (paulstelian97) [reply .travis.yml:62]: Ugh, and that one wasn't updated to 100-character limit, eh.

**Review by paulstelian97 — APPROVED** [review]

**Pull request merged**